src/main/java/de/tum/in/www1/artemis/repository/GradingScaleRepository.java

```diff
@@ -164,6 +164,19 @@ Page<GradingScale> findWithBonusGradeTypeByTitleInCourseOrExamAndUserHasAccessTo
             """)
     Page<GradingScale> findWithBonusGradeTypeByTitleInCourseOrExamForAdmin(@Param("partialTitle") String partialTitle, Pageable pageable);
 
+    /**
+     * Find grading scales for courses
+     *
+     * @param courseIds the courses to which the grading scales belong
+     * @return a set of grading scales for the courses
+     */
+    @Query("""
+                SELECT gradingScale
+                FROM GradingScale gradingScale
+                WHERE gradingScale.course.id IN :courseIds
```

Comment:
To be consistent with the previous queries:
```suggestion
                SELECT gs
                FROM GradingScale gs
                WHERE gs.course.id IN :courseIds
```